### DISCUSSION TITLE
About log

### DISCUSSION THREAD

**Discussion by Iamsohandsome** [victornpb/undiscord#235] [1. Questions & Answers] [answered]
Ummm, i looked through the codes and to me there seems nothing malicious.  But since I am not a pro, I am still a bit worried.

Especially about the "log area".

From a beginner's point of view, I think an option of choosing whether to print the content of the deleted messages on the log can be added.  This may potentially ease worries of noobs like me.

This sounds stupid though(⊙ω⊙)

**Comment by Toby222** [comment, marked as answer]
Screenshot mode hides private information from the log window
That being said the log window doesn't get saved anywhere

**Reply by Iamsohandsome** [reply]
thanks for the information
